soc/nxp/imx/imx9/imx943/Kconfig.defconfig.mimx94398.m33

```diff
@@ -30,4 +30,7 @@ config ETH_NXP_IMX_MSGINTR
 config CACHE_MANAGEMENT
 	default y
 
+config SCMI_CHAN_SEM_TIMEOUT_USEC
+	default 50000
```

Comment:
oh wow, this is quite a jump from 500 us to 50000. I'm curious about the context of the problem.